Makefile.in

```diff
@@ -284,8 +284,8 @@ DIST_SUBDIRS = . test
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/SDL2_image.pc.in \
 	$(srcdir)/SDL2_image.spec.in \
 	$(srcdir)/sdl2_image-config-version.cmake.in \
-	$(srcdir)/sdl2_image-config.cmake.in ar-lib compile \
```

Comment:
Irrelevant to this PR, but this particular removal is correct: We have no `ar-lib` in SDL_image source tree. If someone's autogen.sh run is really adding it somehow, then it should be `git add`ed. @slouken: hint hint..